File: Search/python/02_새.py
# fast campus 강의
# https://www.acmicpc.net/problem/1568
# 2

N = int(input())

index = 1
result = 0
while N > 0:
    if index > N:
        index = 1
    N -= index
    index += 1
    result += 1
print(result)

File: Search/c++/02_새.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/1568
// 2

#include <iostream>

using namespace std;

int N, idx = 1, result;

int main()
{
    cin >> N;

    while (N > 0) {
        if (idx > N)
            idx = 1;
        N -= idx;
        idx++;
        result++;
    }
    cout << result;
} 